inference-engine/tests/functional/plugin/shared/src/single_layer_tests/mat_mul.cpp

```diff
@@ -31,14 +46,32 @@ std::string MatMulTest::getTestCaseName(const testing::TestParamInfo<MatMulLayer
 void MatMulTest::SetUp() {
     InferenceEngine::SizeVector inputShape0;
     InferenceEngine::SizeVector inputShape1;
+    MatMulParams::InputLayerType secondaryInputType;
     auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(netPrecision, inputShape0, inputShape1, targetDevice) = this->GetParam();
+    std::tie(netPrecision, inputShape0, inputShape1, secondaryInputType, targetDevice) = this->GetParam();
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-    auto params = ngraph::builder::makeParams(ngPrc, {inputShape0, inputShape1});
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape0});
+
+    std::shared_ptr<ngraph::Node> secondaryInput;
+    switch (secondaryInputType) {
+        case MatMulParams::InputLayerType::CONSTANT: {
```

Comment:
this switch is similar to the switch which is usied in  `Eltwise`. Could we create ngraph builder (like `makeConv`) for this case?